api/v1alpha1/envoygateway_metrics_types.go

```diff
@@ -24,8 +24,26 @@ type EnvoyGatewayMetricSink struct {
 	// OpenTelemetry defines the configuration for OpenTelemetry sink.
 	// It's required if the sink type is OpenTelemetry.
 	OpenTelemetry *EnvoyGatewayOpenTelemetrySink `json:"openTelemetry,omitempty"`
+	// ExporterInterval configures the intervening time between exports for a
```

Comment:
why are these setting not inside `OpenTelemetry` ?

Reply:
The actual type of OpenTelemetry is `EnvoyGatewayOpenTelemetrySink`, I think it would be more appropriate to configure Sink related. The Exporter feature actually focuses on the server side, which is not suitable for setting in OpenTelemetry. WDYT?

Reply:
what is the distinction b/w sink and exporter here ?

I see you have added these fields into `metricsSink` below

Reply:
Maybe my idea is wrong and I will change it

Reply:
Hi, @arkodg 
I'm sorry that this PR has been delayed for so long due to personal reasons. I have refactored the field into `OpenTelemetry`. Please review again, thank you! :)

Reply:
np, thanks for taking this to completion, lets try to get this into v1.1 !